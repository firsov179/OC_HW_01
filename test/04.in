  if ((fd[1] = open(argv[2], O_WRONLY)) < 0) {
            printf("Can\'t open output file\n");
            exit(-1);
        }
        char res[maxSize];
        for (int i = 0; i < n; ++i) {
            char cur[20];
            sprintf(cur, "%c: %d\n", chars[i], count[i]);
            sprintf(res, "%s%s", res, cur);
        }
        int size = 0;
        while (res[size] != 0) {
            size++;
        }
        if (write(fd[1], res, size) == -1) {
            printf("Can\'t read input file\n");
            exit(-1);
        }
        close(fd[0]);
        close(fd[1]);
    } else {