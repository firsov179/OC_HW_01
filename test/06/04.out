 : 236
i: 25
f: 13
(: 15
d: 6
[: 10
1: 6
]: 10
=: 6
o: 7
p: 8
e: 23
n: 16
a: 8
r: 19
g: 1
v: 1
2: 2
,: 9
O: 2
_: 1
W: 1
R: 1
N: 1
L: 1
Y: 1
): 15
<: 2
0: 6
{: 5
t: 15
": 8
C: 2
\: 5
': 2
u: 7
l: 6
;: 14
x: 3
-: 3
}: 5
c: 10
h: 4
s: 17
m: 1
S: 1
z: 5
+: 4
%: 4
:: 1
w: 2
!: 1
